### PR TITLE
Add a guideline for providing a comment with override = TRUE

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
I'm at work on lunch break and I'm waiting on a 20 minute compile time, so I touch up code guidelines.

This is something that I've asked people to do for a bit and that we have some instances in the codebase of. Shiz reminded me of this but this isn't a diss on him I promise :smile:

Couldn't think of a good example, it's really rare you actually want to override. The instances I could find are complicated elements that would be inappropriate to put in a documentation file like this.

@tgstation/commit-access

### CODE REVIEW

**Comment by Mothblocks** [comment]
It's been 60 seconds and 2 maintainers already liked this you guys need to chill

**Comment by Mothblocks** [comment]
im just teasing i thought the small timeframe was funny

File: .github/guides/STANDARDS.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ While we normally encourage (and in some cases, even require) bringing out of da
 
 * Files and path accessed and referenced by code above simply being #included should be strictly lowercase to avoid issues on filesystems where case matters.
 
-### Signal Handlers
+### RegisterSignal() - Signal Handlers
```
Comment (LemonInTheDark) [inline .github/guides/STANDARDS.md:100]: ?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: .github/guides/STANDARDS.md (hunk 1 above)
Comment (Mothblocks) [reply .github/guides/STANDARDS.md:100]: Just figured I'd do it since they're so close to each other

Comment (LemonInTheDark) [reply .github/guides/STANDARDS.md:100]: If you're gonna make like a category, why not do 
```
### RegisterSignal()

#### Signal Handlers

#### Signal Override
```
Or is that just my compulsion kicking in

Comment (Mothblocks) [reply .github/guides/STANDARDS.md:100]: Sure

**Review by LemonInTheDark — APPROVED** [review]

**Pull request merged**